README.md

```diff
@@ -62,7 +74,67 @@ The component has five different exported consumption APIs:
 - `withResponsive`: A HoC which passes the responsive data to the `responsive` prop
 - `withIsMobile`: A HoC which passes `isMobile` and `isCalculated` props only
 
-### Rendering components with `useResponsive` hook. (Preferred method)
+
+### How setup the package
```

Comment:
How to setup

Reply:
Done thanks 👍

Reply:
Now it just says "How setup", I was just saying to add the "to"

Reply:
Double checked now and its fixed 🙏